plivo/resources/messages.py

```diff
@@ -101,7 +113,13 @@ def get(self, message_uuid):
         powerpack_id=[optional(of_type(six.text_type))],
         tendlc_campaign_id=[optional(of_type(six.text_type))],
         destination_country_iso2=[optional(of_type(six.text_type))],
-        tendlc_registration_status=[optional(of_type(six.text_type))])
+        tendlc_registration_status=[optional(of_type(six.text_type))],
+        message_type=[
+            optional(all_of(of_type(six.text_type), is_in(('sms', 'mms', 'whatsapp'))))],
+        conversation_id=[optional(of_type(six.text_type))],
+        conversation_origin=[optional(all_of(of_type(six.text_type), is_in(('user_initiated', 'business_initiated', 'service', 'utility', 'authentication', 'marketing'))))]
```

Comment:
As discussed let's not have `user_initiated, business_intiated` conversation types and keep it up to date with the conversation types supported by WhatsApp.